staging/src/k8s.io/component-base/metrics/options.go

```diff
@@ -56,12 +57,23 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 			"The format is <major>.<minor>, e.g.: '1.16'. "+
 			"The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics, "+
 			"rather than being surprised when they are permanently removed in the release after that.")
+	fs.StringSliceVar(&o.DisabledMetrics,
+		"disabled-metrics",
+		o.DisabledMetrics,
+		"This flag provides an escape hatch for misbehaving metrics. "+
+			"You must provide the fully qualified metric name in order to disable it.")
 }
 
 // Apply applies parameters into global configuration of metrics.
 func (o *Options) Apply() {
 	if o != nil && len(o.ShowHiddenMetricsForVersion) > 0 {
 		SetShowHidden()
+		// set disabled metrics
+		disabledMetricsLock.Lock()
+		defer disabledMetricsLock.Unlock()
+		for _, metricName := range o.DisabledMetrics {
+			SetDisabledMetric(metricName)
```

Comment:
Does this need any validation?

Reply:
I can patch the test to use this path.

Reply:
ah good call, there was a bug that I caught going through this path.

Reply:
yessss